bootstrap.sh

```diff
@@ -8,6 +8,9 @@ export TEMP_DIR=/tmp
 export TMPDIR=/tmp
 export TMP_DIR=/tmp
 
+# install expect
+apt-get install -y expect
```

Comment:
assume it's been done elsewhere then, default ubuntu install seems to require that (for me)